src/dotnet/Azure.SDK.ChangelogGen/Azure.SDK.ChangelogGen/Azure.SDK.ChangelogGen.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
If this is a stand-alone tool intended to be run locally, it should be under the `$/tools` path.

Reply:
sure, will move to .../tool.